README.md

```diff
@@ -54,6 +54,11 @@ module "bigquery" {
       billable = "true"
       owner    = "joedoe"
     },
+    materialized_view = {
```

Comment:
This is no longer in `tables`, you can probably just remove this from the README example entirely.

Reply:
Oh, good catch. I was under the impression that this was generated and forgot to double check that it was gone after regenerating the docs.